src/Controller/DashboardController.php

```diff
@@ -79,27 +79,26 @@ public function dashboard(Request                     $request,
                 return $this->redirectToRoute('no_team');
             }
         }
-        $teamPath = $teamRepository->getPath($currentTeam);
 
         $audit = $auditRepository->findAllByTeam($currentTeam);
-        $daten = $transferRepository->findActiveTransfersByTeamPath($teamPath);
-        $av = $transferRepository->findActiveOrderProcessingsByTeamPath($teamPath);
-        $vvt = $procedureRepository->findActiveByTeamPath($teamPath);
-        $vvtDsfa = $impactAssessmentRepository->findActiveByTeamPath($teamPath);
-        $kontakte = $contactRepository->findActiveByTeamPath($teamPath);
-        $tom = $tomRepository->findActiveByTeamPath($teamPath);
+        $daten = $transferRepository->findActiveTransfersByTeam($currentTeam);
+        $av = $transferRepository->findActiveOrderProcessingsByTeam($currentTeam);
+        $processes = $processRepository->findActiveByTeam($currentTeam);
+        $vvtDsfa = $impactAssessmentRepository->findActiveByTeam($currentTeam);
+        $contacts = $contactRepository->findActiveByTeam($currentTeam);
+        $tom = $tomRepository->findActiveByTeam($currentTeam);
         $forms = $formRepository->findPublicByTeam($currentTeam);
-        $policies = $policyRepository->findPublicByTeamPath($teamPath);
-        $software = $softwareRepository->findActiveByTeamPath($teamPath);
+        $policies = $policyRepository->findPublicByTeam($currentTeam);
+        $software = $softwareRepository->findActiveByTeam($currentTeam);
         $tasks = $taskRepository->findActiveAndOpenByTeam($currentTeam);
         $loeschkonzepte = $deletionConceptRepository->findByTeam($currentTeam);
         $vvtdatenkategorien = $dataCategoryRepository->findByTeam($currentTeam);
         $kritischeAudits = $auditRepository->findCriticalByTeam($currentTeam);
-        $kritischeVvts = $procedureRepository->findCriticalByTeamPath($teamPath);
+        $criticalProcesses = $processRepository->findCriticalByTeam($currentTeam);
         $openDsfa = $impactAssessmentRepository->findActiveAndOpenByTeam($currentTeam);
         $buchungen = $bookingRepository->findActiveByUser($user);
 
-        $assignVvt = $procedureRepository->findActiveByTeamAndUser($currentTeam, $user);
+        $assignVvt = $processRepository->findActiveByTeamAndUser($currentTeam, $user);
```

Comment:
The counter for the single instances are a bit inconsistent in the moment, when I check dashboard of a child team that is inheriting instances from a parent team, but disabled the use of inherited instances:

![image](https://github.com/H2-invent/open-datenschutzcenter/assets/47010475/6c5273ed-3c34-44a6-aa0e-930d88db29e9)

Here just one vvt is inherited from the parent. This vvt was disabled by the child team so the counter is 0 despite the vvt is listed at `/vvt`. On the other hand the datenweitergabe and the auftragsverarbeitungs object are connected to the disabled vvt, but here the counter is correct.

![image](https://github.com/H2-invent/open-datenschutzcenter/assets/47010475/787cab8d-9414-4597-a02b-08d86d810d5f)

I think the expected behavior would be to count any objects independent of active or not. But at least it should be consistent.

Reply:
I think the expected behavior would be for the ignored elements to not be counted - we wouldn't want to clutter the dashboard with unused elements, right? But if the users see this differently, can easily be changed later.

The behavior is now consistent for all elements except the last three, as the inheritance is not yet defined for those. We should probably discuss that in the next meeting, but I think for now it is ok.